src/plugins/visible-range/index.js

```diff
@@ -126,6 +137,38 @@ export default class VisibleRangePlugin extends AbstractPlugin {
     }
   }
 
+  /**
+   * @summary Gets the latitude range defined by the viewer's panoData
+   * @returns {double[]|null}
+   * @private
+   */
+  getPanoLatitudeRange() {
+    const p = this.psv.prop.panoData;
+    if (p.croppedHeight === p.fullHeight && p.croppedY === 0) {
+      return null;
+    }
+    else {
+      const latitude = y => Math.PI * (y / p.fullHeight) - (Math.PI / 2);
```

Comment:
I think the correct transformation is
```suggestion
      const latitude = y => Math.PI * (1 - y / p.fullHeight) - (Math.PI / 2);
```

if fullHeight = 2000px then
```
0px = PI/2
2000px = -PI/2
```